lib/howl/ui/editor.moon

```diff
@@ -169,21 +169,20 @@ class Editor extends PropertyObject
     set: (v) => @view.config.view_line_padding = v
 
   @property line_wrapping:
-    get: =>
-      -- sci_val = @sci\get_wrap_mode!
-      -- switch sci_val
-      --   when Scintilla.SC_WRAP_NONE then 'none'
-      --   when Scintilla.SC_WRAP_WORD then 'word'
-      --   when Scintilla.SC_WRAP_CHAR then 'character'
-      --   else '(unknown)'
+    get: => @view.config.view_line_wrap
+    set: (value) =>
+      unless value\umatch(r'^(?:none|word|character)$')
```

Comment:
Particular reason for the parens here? Seems slightly inconsistent with the rest of the file.